sssoja/tv-bland
# creates show view page

### What changes have you made?
- Created a basic show view layout 
- Created a `show` route under pages which can be accessed via the `shows` directory 
- Abstracted the bulk of the placeholder content into the `ShowData` component
- Added dummy `Show` data in the `shows` object of the `ShowList` component 
- Installed dependencies for font awesome to set up the star rating functionality

### Which issue(s) does this PR fix?
Fixes #3 

### Screenshots (if there are design changes)
<img width="894" alt="Screenshot 2020-11-21 at 22 21 10" src="https://user-images.githubusercontent.com/53219789/99887813-f9ca0a80-2c47-11eb-84f0-ed4bc64df5df.png">

<img width="271" alt="Screenshot 2020-11-21 at 22 21 38" src="https://user-images.githubusercontent.com/53219789/99887815-fdf62800-2c47-11eb-9109-9a78b22f6a6f.png">

### How to test
Run the server on `localhost/3000` and check that you can reach `localhost/3000/shows/show` via the homepage by clicking on _any_ of the show images on the homepage:
- check that the URL route in the browser displays as `http://localhost:3000/shows/show`
- ensure that the user can navigate back to the homepage
